redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,345 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crwoperator.v1.1.0
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion":"org.eclipse.che/v1",
+          "kind":"CheCluster",
+          "metadata":{
+             "name":"codeready"
+          },
+          "spec":{
+             "server":{
+                "cheFlavor":"codeready",
+                "cheImage": "quay.io/crw/server-container",
+                "cheImageTag": "1.1-45",
+                "tlsSupport":false,
+                "selfSignedCert":false
+             },
+             "database":{
+                "externalDb":false,
+                "chePostgresHostname":"",
+                "chePostgresPort":"",
+                "chePostgresUser":"",
+                "chePostgresPassword":"",
+                "chePostgresDb":""
+             },
+             "auth":{
+                "openShiftoAuth":false,
+                "externalKeycloak":false,
+                "keycloakURL":"",
+                "keycloakRealm":"",
+                "keycloakClientId":""
+             },
+             "storage":{
+                "pvcStrategy":"common",
+                "pvcClaimSize":"1Gi",
+                "preCreateSubPaths": true
+             }
+          }
+        }
+      ]
+    certified: "true"
+    categories: "Developer Tools"
+    containerImage: quay.io/crw/operator-container:1.1-7
```

Comment:
Is this image built by Red Hat's build pipeline today? If not, this operator should be added to the `community-operators` directory instead of `redhat-operators`

Reply:
@SamiSousa I will replace this image with https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/codeready-workspaces/server-operator after 1.1 tag is out which is going to happen mid April.

Reply:
@SamiSousa I have updated image reference. It's a RHCC image now.

Reply:
@SamiSousa can you please review again? Let me know if I missed any comments or feedback.